### PR TITLE
Werkings gebieden endpoints voor all_valid en valid

### PR DESCRIPTION
**Pull request by JordyMoos**
werkingsgebieden are now no longer filtered by something like `status = 'Vigerend'` but needs to be used by a valid maatregel or valid beleidskeuze.

### CODE REVIEW

File: Api/Models/onderverdeling.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Onderverdeling(CommonMixin, db.Model):
     symbol = Column(Unicode(265)) # @todo: length feels like a typo
     Werkingsgebied = Column(Unicode(256))
     UUID_Werkingsgebied = Column(ForeignKey('Werkingsgebieden.UUID'), nullable=False)
-    SHAPE = Column(Geometry, nullable=False)
+    SHAPE = deferred(Column(Geometry, nullable=False))
```
Comment (JordyMoos) [inline Api/Models/onderverdeling.py:18]: `deferred` tells sqlalchemy to not fetch this field unless explicitly requested

File: Api/Endpoints/werkingsgebieden_data_manager.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                        w.ID,
+                        w.UUID,
+                        w.Begin_Geldigheid,
+                        w.Eind_Geldigheid,
+                        w.Created_Date,
+                        w.Modified_Date,
+                        w.Created_By,
+                        w.Modified_By,
+                        w.Werkingsgebied,
+                        w.symbol
```
Comment (JordyMoos) [inline Api/Endpoints/werkingsgebieden_data_manager.py:25]: In the new views we do not fetch the `SHAPE` field which is of type geometry, as it is not supported by the mssql driver

File: Tests/test_api.py [hunk 3]
```diff
@@ -26,6 +26,67 @@
...
 class TestApi:
 
+    def test_werkingsgebied_all_valid_view(self, client, client_fred):
+        response = client_fred.get("v0.1/valid/werkingsgebieden")
+        assert response.status_code == 200, f"Status code was {response.status_code}"
+        data = response.get_json()
+        assert len(data) >= 2, f"Expecting at least 2 results"
+
+        expected_uuids = set([
+            "8EB1ED00-0002-2222-0000-000000000000",
```
Comment (Swendude) [inline Tests/test_api.py:36]: Love this UUID choice

**Review by Swendude — APPROVED** [review]
Looking good, feel free to merge!

**Pull request merged**